.env.example

```diff
@@ -0,0 +1,3 @@
+#pre-auth 
+PAYWAY_V2_COMPLETE_PRE_AUTH_PATH=""
+PAYWAY_V2_CANCEL_PRE_AUTH_PATH=""
```

Comment:
@panhachom what do you think if we rename to this to make it more semantic
```
PAYWAY_V2_PRE_AUTH_COMPLETE_PATH
PAYWAY_V2_PRE_AUTH_CANCEL_PATH
```

Reply:
how about this bong @channainfo 
```
COMPLETE_PRE_AUTH_PATH 
CANCEL_PRE_AUTH_PATH
```